sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -122,36 +122,41 @@ export type AcsSmsReceivedEventData = AcsSmsEventBase & {
 };
 
 // @public
-export type AppAction = "Restarted" | "Stopped" | "ChangedAppSettings" | "Started" | "Completed" | "Failed";
+export type AppAction = string;
 
 // @public
 export interface AppConfigurationKeyValueDeletedEventData {
     etag?: string;
     key?: string;
     label?: string;
+    syncToken?: string;
```

Comment:
Why do we have this new property?

Reply:
It was added with this commit: https://github.com/Azure/azure-rest-api-specs/pull/11819 2 months back. The swagger is pointing to master branch. So, it is picking up new changes. The changes seemed minimal. So, I took them it

Reply:
Okay, this is not a breaking change since the new properties are optional, however I would like @ellismg to confirm if we want to start exposing these properties now. Other than this I think the changes look reasonable and all checks pass

Reply:
Exposing these now is great!